### DISCUSSION TITLE
The OffSet Property Not working

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5696] [General]
<img width="905" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/119593989/210a5186-be24-4e8f-a59a-7091b62dfe54">

I have enabled the options for that but visually i didnt see any changes. is that deprecated ?

**Comment by artf** [comment]
with `showOffsetsSelected` it shows offsets (margins/paddings) when you hover selected components, without that option, the selected component never shows them.
I'd say it's quite a low-value config, it would make sense one day to do a proper canvas spot for that functionality with more specific configurable options.
